libs/platform/src/lib/components/value-help-dialog/value-help-dialog/value-help-dialog.component.ts

```diff
@@ -0,0 +1,643 @@
+      backdropClickCloseable: false,
+      hasBackdrop: true,
+      verticalPadding: false,
+      escKeyCloseable: false,
+      dialogPanelClass: `fdp-value-help-dialog ${this.mobile ? 'fdp-value-help-dialog-mobile' : ''}`,
+      ...this.dialogConfig,
+
+      fullScreen: this.mobile,
+      maxWidth: this.mobile ? null : '92%',
+      width: this.mobile ? null : '1080px',
+      height: this.mobile ? null : '98%',
+    });
+    this._listenDialogEvents();
+  }
+
+  /**
+   * Toggle panel with selected and excluded items
+   */
+  toggleSelectedPanel(): void {
+    const selectedCount = this.selectedAndIncluded !== 0;
+    const excludedCount = this.validExcludedItems.length !== 0;
+
+    if (selectedCount || excludedCount) {
+      this._selectedExpandState = !this._selectedExpandState;
+    }
+  }
+
+  /**
+   * Toggle advanced search panel
+   */
+  toggleAdvancedSearch(state?: boolean): void {
+    if (typeof state !== 'undefined') {
+      this.isOpenAdvanced = state;
+    } else {
+      this.isOpenAdvanced = !this.isOpenAdvanced;
+    }
+
+    if (this.mobile) {
+      this.selectedTab = this.isOpenAdvanced ? VhdTab.advancedSearch : VhdTab.selectFromList;
+    }
+  }
+
+  /**
+   * Clear all selected items
+   */
+  clearSelectedItems(): void {
+    this.value.selected = [];
+    this.value.included = [];
+    this._changeDetectorRef.markForCheck();
+  }
+
+  /**
+   * Clear all excluded items
+   */
+  clearExcludedItems(): void {
+    if (this.validExcludedItems.length) {
+      this.value.excluded = [];
+      this._changeDetectorRef.markForCheck();
+    }
+  }
+
+  /**
+   * Bacis search by all filled filters
+   */
+  search(): void {
+    const nonEmptyFilters = new Map();
+    this._displayedFilters.filter(({ value }) => !!value && value.trim().length).forEach(filter => {
+      nonEmptyFilters.set(filter.key, filter.value);
+    });
+
+    if (this._mainSearch.length) {
+      nonEmptyFilters.set('*', this._mainSearch);
+    }
+    this.dataSource.match(nonEmptyFilters);
+  }
+
+  /**
+   * Apply search from advanced filters view, using when mobile view is active
+   */
+  searchAdvanced(): void {
+    this.search();
+    this.selectedTab = VhdTab.selectFromList;
+    this.isOpenAdvanced = false;
+  }
+
+  /** Remove item from selected array */
+  removeSelected(index: number): void {
+    this.value.selected = this.selectedItems.filter((_, i) => i !== index);
+    this._changeDetectorRef.markForCheck();
+  }
+
+  /**
+   * Remove included items
+   */
+  removeIncluded(index: number): void {
+    this.value.included = this.includedItems.filter((_, i) => i !== index);
+    this._changeDetectorRef.markForCheck();
+  }
+
+  /**
+   * Remove excluded items
+   */
+  removeExcluded(index: number): void {
+    this.value.excluded = this.excludedItems.filter((_, i) => i !== index);
+    this._changeDetectorRef.markForCheck();
+  }
+
+  /**
+   * Switch tab by type
+   */
+  switchTab(type?: VhdTab): void {
```

Comment:
Let's use `(type: VhdTab = null)` to keep it consistent with initial `null` value